src/DependencyInjection/CompilerPass/MakeCommandRegistrationPass.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Symfony\Bundle\MakerBundle\DependencyInjection\CompilerPass;
+
+use Symfony\Bundle\MakerBundle\Command\MakerCommand;
+use Symfony\Bundle\MakerBundle\MakerInterface;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Reference;
+
+class MakeCommandRegistrationPass implements CompilerPassInterface
+{
+    const MAKER_TAG = 'maker.command';
```

Comment:
Shouldn't we use explicit `public const`?

Reply:
Why not? :) I've done it over on #59